generators/client/templates/angular/webpack/webpack.common.js.ejs

```diff
@@ -92,9 +92,8 @@ module.exports = (options) => ({
             }
         }),
         new CopyWebpackPlugin([
-            { from: './node_modules/swagger-ui/dist/css', to: 'swagger-ui/dist/css' },
-            { from: './node_modules/swagger-ui/dist/lib', to: 'swagger-ui/dist/lib' },
-            { from: './node_modules/swagger-ui/dist/swagger-ui.min.js', to: 'swagger-ui/dist/swagger-ui.min.js' },
+            { from: './node_modules/swagger-ui-dist/*.{js,css,html,png}', to: 'swagger-ui', flatten: true, ignore: 'index.html'},
+            { from: './node_modules/axios/dist/axios.min.js', to: 'swagger-ui'},
```

Comment:
axios is present in the node_modules directory for both angular and react, but it is explicitly declared only in the react app. Should I add an explicit dependency in the package.json of the angular app?